engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/K8sDeploymentManager.scala

```diff
@@ -158,6 +160,8 @@ object K8sDeploymentManager {
 
   val defaultParallelism = 1
 
+  val nussknackerInstanceNameLabel: String = "nussknacker.io/nussknackerInstanceName"
```

Comment:
It should be `app.kubernetes.io/instance` take a look at: https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/